hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java

```diff
@@ -250,7 +249,7 @@ private static BucketCache createBucketCache(Configuration c) {
           bucketCacheSize, blockSize, bucketSizes, writerThreads, writerQueueLen, persistentPath,
           ioErrorsTolerationDuration, c);
     } catch (IOException ioex) {
-      LOG.error("Can't instantiate bucket cache", ioex); throw new RuntimeException(ioex);
```

Comment:
what does the log look like when this failure bubbles up the call stack? (I would like an answer before this merges)